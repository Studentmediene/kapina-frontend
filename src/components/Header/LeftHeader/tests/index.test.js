import React from 'react';
import { shallow } from 'enzyme';

import { LeftHeader } from '../';

describe('<LeftHeader />', () => {
  it('renders correctly', () => {
    const tree = shallow(<LeftHeader />);
    expect(tree).toMatchSnapshot();
  });
});
