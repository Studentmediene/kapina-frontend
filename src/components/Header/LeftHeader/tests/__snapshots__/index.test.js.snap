// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<LeftHeader /> renders correctly 1`] = `
<Fragment>
  <div
    className="largeNavContainer"
  >
    <NavList
      links={
        Array [
          Object {
            "path": "/programmer",
            "title": "Programmer",
          },
          Object {
            "path": "/sendeplan",
            "title": "Sendeplan",
          },
        ]
      }
    />
  </div>
  <div
    className="smallNavContainer"
  >
    <HamburgerMenu />
  </div>
</Fragment>
`;
